src/Security.tsx

```diff
@@ -33,7 +33,12 @@ const Security: React.FC<{
     return oktaAuth.authStateManager.getAuthState();
   });
   const [oktaAuthMajorVersion] = React.useState(() => {
-    const majorVersion = oktaAuth?.userAgent?.split('/')[1]?.split('.')[0];
+    if (!oktaAuth || !oktaAuth._oktaUserAgent) {
+      return null;
+    }
+
+    const [oktaAuthVersion] = Object.values(oktaAuth._oktaUserAgent.getHttpHeader());
```

Comment:
https://github.com/okta/okta-auth-js/blob/master/lib/OktaUserAgent.ts#L34

You can use the `getVersion` method.